src_c/draw.c

```diff
@@ -946,12 +956,15 @@ static int
 clip_and_draw_line_width(SDL_Surface *surf, SDL_Rect *rect, Uint32 color,
                          int width, int *pts)
 {
-    int loop;
     int xinc = 0, yinc = 0;
-    int newpts[4];
-    int range[4]; /* {xmin, ymin, xmax, ymax} */
+    int bounding_rect[4];
+    bounding_rect[0] = INT_MAX;
+    bounding_rect[1] = INT_MAX;
+    bounding_rect[2] = 0;
+    bounding_rect[3] = 0;
+    int original_values[4];
```

Comment:
You have to move type declarations above assignments for the old MSVC compiler that python2.7 uses.